.gitignore

```diff
@@ -8,4 +8,4 @@ Berksfile.lock
 .ruby-version
 .ruby-gemset
 .rspec
-
+.idea
```

Comment:
I prefer to keep this .gitignore file small, and would recommend you to put .idea in your ~/.gitignore_global file.